lib/live_debugger/app/debugger/callback_tracing/web/hook_components/filters_fullscreen.ex

```diff
@@ -62,19 +64,35 @@ defmodule LiveDebugger.App.Debugger.CallbackTracing.Web.HookComponents.FiltersFu
 
     ~H"""
     <div class="flex">
-      <.button
-        variant="secondary"
-        aria-label="Open filters"
-        size="sm"
-        class={["flex gap-1", if(@applied_filters_number > 0, do: "rounded-r-none")]}
-        phx-click="open-filters"
+      <TraceSettings.maybe_add_tooltip
+        display_mode={@display_mode}
+        id="filters-tooltip"
+        content="Filters"
+        position="top-center"
       >
-        <.icon name="icon-filters" class="w-4 h-4" />
-        <span class={["ml-1", @label_class]}>Filters</span>
-        <span :if={@applied_filters_number > 0}>
-          (<%= @applied_filters_number %>)
-        </span>
-      </.button>
+        <.button
+          variant="secondary"
+          aria-label="Open filters"
+          size="sm"
+          class={[
+            "flex gap-1",
+            if(@applied_filters_number > 0, do: "rounded-r-none"),
+            @label_class,
+            @display_mode == :dropdown && "!w-full !border-none text-primary-text"
```

Comment:
Why these buttons have `text-primary-text`, meaning they're black? Imo they should be dark blue like everything else

<img width="430" height="237" alt="Screenshot 2025-11-17 at 09 25 54" src="https://github.com/user-attachments/assets/f00ba6f7-79e5-41fb-a6b0-594dd3b8bab6" />